src/smc-webapp/projects_nav.cjsx

```diff
@@ -289,7 +289,7 @@ DropdownProjectsNav = rclass
         else
             title = "Open projects"
 
-        <Nav className='smc-dropdown-projects' style={display:'flex', margin:'0', flex:'1', fontSize:'25px', textAlign:'center', padding:'15px'}>
+        <Nav className='smc-dropdown-projects' style={display:'flex', margin:'0', flex:'1', fontSize:'25px', textAlign:'center', padding:'5px'}>
```

Comment:
funny, you have no way of knowing, but I thought I broke that in mobile in my cocalc branch. Hence that's how my fix looks like: https://github.com/sagemathinc/smc/blob/cocalc/src/smc-webapp/projects_nav.cjsx#L297